trio/_core/_run.py

```diff
@@ -447,13 +472,7 @@ def __enter__(self):
             task._activate_cancel_status(self._cancel_status)
         return self
 
-    def _exc_filter(self, exc):
-        if isinstance(exc, Cancelled):
-            self.cancelled_caught = True
-            return None
-        return exc
-
-    def _close(self, exc):
+    def _close(self, exc, collapse=True):
```

Comment:
The default for collapse should be false, so that passing through a normal CancelScope (that is not attached to a nursery) doesn't strip ExceptionGroup layers.

Reply:
It only affects MultiErrors, and if I did this, `test_cancel_scope_multierror_filtering()` would start failing, as it **expects** the `MultiError` passing through the cancel scope to be collapsed into the contained `KeyError`.